modules/sandboxed-containers-common-terms.adoc

```diff
@@ -0,0 +1,34 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/understanding_sandboxed_containers.adoc
+
+[id="sandboxed-containers-common-terms_{context}"]
+
+= Common terms
+
+.Sandbox
+A sandbox is a controlled environment where programs can run. Within the controlled environment, you can execute untested or untrusted programs without risking harm to the host machine or the operating system. A sandbox keeps your workloads isolated from the rest of the system.
```

Comment:
>! execute _verb_
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.